src/NServiceBus.Core.Tests/Unicast/StartAndStoppablesRunnerTests.cs

```
@@ -141,6 +142,17 @@ public void Should_throw_if_startable_fails_asynchronously()
             Assert.True(startable2.Stopped);
         }
 
+        [Test]
+        public void Should_throw_friendly_invalidoperationexception_when_start_returns_null()
```

Errors from Stop are swallowed and logged, so it is not testable.